packages/contracts-bedrock/hardhat.config.ts

```diff
@@ -35,6 +35,11 @@ const config: HardhatUserConfig = {
         'ac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80',
       ],
     },
+    hivenet: {
+      chainId: Number(process.env.CHAIN_ID),
```

Comment:
Does hive use a particular chainid or is it nice to be able to configure it? I don't love that this file uses so many env vars, I always find myself commenting out lines because env vars are not defined

Reply:
The L1 RPC env var definitely changes, so I'm inclined to keep the other env vars, as the env is being configured anyway.